src/commands/build.rs

```diff
@@ -1,16 +1,84 @@
 pub async fn build(config: &fpm::Config) -> fpm::Result<()> {
```

Comment:
Right now `build()` is in really poor shape. No one can read the function body and guess what is going on. To understand the body of a function you can not say oh just read the body of every other function, compiler and CPU can do that humans can not. 

Have a separate function to build for regular package and for translation package for example, check in beginning and branch. Some code duplication is okay, but if no one can understand the code that's not okay.